core/src/main/scala/chisel3/Layer.scala

```diff
@@ -49,6 +49,7 @@ object layer {
       case Layer.Root => name
       case _          => s"${parent.fullName}.$name"
     }
+
```

Comment:
Nit I would remove this whitespace

Reply:
Fixed in https://github.com/chipsalliance/chisel/pull/3799/commits/54d826170888f31d1e3116ac674d34d2a3a9005b.